module/VuFind/src/VuFind/Controller/Feature/RequestHelperTrait.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * VuFind Action Feature Trait - Alphabetic browse support
+ * Depends on direct access to the Service Manager.
```

Comment:
These header comments (and the matching ones below) don't apply to this file -- please revise. :-)

Reply:
Oh, forgot to change this.. Will adjust